.github/workflows/build.yml

```diff
@@ -4,30 +4,29 @@ name: Build Java
 
 on:
   push:
-    branches: [ '**' ]
+    branches: [ 'master' ]
```

Comment:
nice catch
who's paying on builds btw?

Reply:
Free for public builds, but they will limit you if you do too much. (I think)